lib/features.js

```diff
@@ -83,5 +83,8 @@ module.exports = {
     generators: false,
 
     // React JSX parsing
-    jsx: false
+    jsx: false,
+
+    // enable parsing of import/export declarations
+    module: false
```

Comment:
True, we could use `extra.scriptType==="module"` everywhere. The way I'm thinking here is to have the `scriptType` to alter some of the feature flags based on its value, but keeping the code clean by verifying the actual feature flags. Although, I will be ok with using scriptType directly for now.